src/pages/authorization/agent-authorization/index.mdx

```diff
@@ -9,7 +9,7 @@ desc: "Learn how to authorize your app to send a sample request using our Messag
 
 # Introduction
 
-To call LiveChat APIs, you need to authorize your calls with an access token. This document describes three authorization flows that let you acquire a token:
+The authorization methods described in this document may be used to create the LiveChat and HelpDesk products. To call LiveChat or HelpDesk APIs, you need to authorize your calls with an access token. This document describes three authorization flows that let you acquire a token:
```

Comment:
tutaj ogólnie przeszłyśmy z Olą przez dokumentację i ustaliłyśmy, że w poniższych miejscach trzeba zrobić zmiany. Póki co takie najszybsze żeby wskazywało, że dane rzeczy tez dotyczą HD. Ewentualne większe zmiany będę musiały być ronione stopniowo in the future.

Reply:
Dokładnie :)